src/Navigation.vue

```diff
@@ -47,32 +56,49 @@ export default {
     }
   },
   computed: {
+    /**
+     * @return {Boolean} Can the slider move forward?
+     */
     canAdvanceForward() {
       return this.parentContainer.canAdvanceForward || false;
     },
+    /**
+     * @return {Boolean} Can the slider move backward?
+     */
     canAdvanceBackward() {
       return this.parentContainer.canAdvanceBackward || false;
     }
   },
   methods: {
+    /**
+     * Trigger page change on +/- 1 depending on the direction
+     * @param {"backward"} [direction]
+     * @return {void}
+     */
     triggerPageAdvance(direction) {
-      if (direction) {
-        this.$parent.advancePage(direction);
-      } else {
-        this.$parent.advancePage();
-      }
+      /**
+       * @event paginationclick
+       * @type {string}
+       */
+      this.$emit("navigationclick", direction);
     }
   }
 };
 </script>
 
-<style scoped>
```

Comment:
I remove scoped attribute, because you have enough specific styles

Reply:
Hmm I think we should leave this as is since it covers some edge cases, like styling multiple sliders differently on one page

Reply:
How it's will be work?
Why not add mixin class on Carousel component and not use something like this
```
.mycarouse .VueCarousel-navigation-button {
  color: red;
}
```

Reply:
@quinnlangille answer plz

Reply:
I agree with you - I just don't want to have it be a breaking change. It could definitely be a change considered for when we roll to `v1.0.0`!

Reply:
@quinnlangille oh, ok. I return this.

Reply:
You're too fast! Thanks @edDimensi, once travis passes below I'll approve this 👍

Reply:
Ok.